docs/docs/troubleshooting.md

```diff
@@ -73,3 +76,88 @@ CONFIG_BT_CTLR_TX_PWR_PLUS_8=y
+
+### MacOS/Windows Dual Booting
+
+If you are utiizing Apple's BootCamp feature or running a hackintosh that allows you to easily dual-boot your system between macOS and Windows, you will find that you can only use your bluetooth devices on one OS or the other without re-pairing. This is because ZMK stores the host address and keys in the profiles only one time. This means that a host can only pair with exactly one ZMK profile.
+
+The workaround for this limitation is as follows:
+
+1. First, boot into Windows and pair your device. This creates the registry entry we will need to edit later on.
+1. Reboot into macOS
+1. In macOS, clear the profile on your keyboard and pair with your computer.
+1. Once paired, hold Option + Control + Shift and click on the bluetooth icon in your menubar. Take note of the "Address" of your computer's bluetooth device (`65-FD-3B-1A-DB-48` here), as well as the address of the connected device (`E6-24-54-08-BC-A3` here).
+
+   | ![BT Contextual Menu](../docs/assets/troubleshooting/dualboot/bt_addresses.png) |
+   | :-----------------------------------------------------------------------------: |
+   |                             Bluetooth Context Menu                              |
+
+1. Now open the terminal app (Applications/Utilities/Terminal.app) and paste the following command and enter your root password.
+
+   <Tabs
+   defaultValue="high_sierra"
+   values={[
+   {label: 'macOS High Sierra 10.13 and newer', value: 'high_sierra'},
+   {label: 'macOS Sierra 10.12 and older', value: 'sierra'},
+   ]}>
+   <TabItem value="high_sierra">
+
+   ```
+   sudo defaults read /private/var/root/Library/Preferences/com.apple.bluetoothd.plist
+   ```
+
+   </TabItem>
+   <TabItem value="sierra">
+
+   ```
+   sudo defaults read /private/var/root/Library/Preferences/blued.plist
+   ```
+
+   </TabItem>
+   </Tabs>
+
+   This will print the contents of the bluetooth plist file containing the pairing keys for your bluetooth devices.
+
+   | ![BT Plist](../docs/assets/troubleshooting/dualboot/plist.png) |
+   | :------------------------------------------------------------: |
+   |                        Bluetooth .plist                        |
+
+1. Look for the section that starts with `SMPDistributionKeys =`. Below this, find your host's bluetooth address (`65-FD-3B-1A-DB-48` in this example) and then look for the address of your keyboard (`E6-24-54-08-BC-A3` in this example).
+1. Under your keyboard address, find the value labeled LocalLTK (in the screenshot above it is `0x8968239e350b2cb7df16d8f47c774e2c`). Copy this value into a text file that you can access from Windows, this is the value we will need to enter into the registry later on.
+1. Reboot into Windows
+1. Once logged in, turn off bluetooth.
+1. Grab a piece of software called psexec (google) and copy the .exe to the desktop. This will allow us to easily open Regedit with elevated privelages.
```

Comment:
Could we make a direct link here rather than saying to google something?

Reply:
I've added direct link in latest push. I don't care where the documentation ends up. I wasn't really sure what the best place for it was. Open to moving it wherever makes sense.